src/main/java/frc/robot/subsystems/leds/LEDSubsystem.java

```diff
@@ -88,14 +90,23 @@ public Command setRunAlongCmd(
         });
   }
 
-  public Command defaultStateDisplayCmd(BooleanSupplier enabled, BooleanSupplier targetIsSpeaker) {
+  public Command defaultStateDisplayCmd(BooleanSupplier enabled, Supplier<Target> target) {
     return Commands.either(
-            Commands.either(
-                this.setBlinkingCmd(new Color("#ffff00"), new Color(), 10.0)
-                    .until(() -> !targetIsSpeaker.getAsBoolean() || !enabled.getAsBoolean()),
-                this.setBlinkingCmd(new Color("#ff7777"), new Color(), 10.0)
-                    .until(() -> targetIsSpeaker.getAsBoolean() || !enabled.getAsBoolean()),
-                targetIsSpeaker),
+            Commands.select(
+                Map.of(
+                    Target.SPEAKER,
+                    this.setBlinkingCmd(new Color("#ffff00"), new Color(), 10.0)
```

Comment:
good colors!

Reply:
Love the visual confirmation of what target has been selected 👍